server/sonar-server/src/test/java/org/sonar/server/user/UserUpdaterTest.java

```diff
@@ -568,12 +593,21 @@ public void update_user() throws Exception {
     assertThat(dto.getCryptedPassword()).isNotEqualTo("650d2261c98361e2f67f90ce5c65a95e7d8ea2fg");
     assertThat(dto.getCreatedAt()).isEqualTo(1418215735482L);
     assertThat(dto.getUpdatedAt()).isEqualTo(1418215735486L);
+
+    List<SearchHit> indexUsers = es.getDocuments(UserIndexDefinition.INDEX, UserIndexDefinition.TYPE_USER);
+    assertThat(indexUsers).hasSize(1);
+    assertThat(indexUsers.get(0).getSource())
+      .contains(
+        entry("login", "marius"),
+        entry("name", "Marius2"),
+        entry("email", "marius2@mail.com"));
   }
 
   @Test
   public void update_user_with_scm_accounts_containing_blank_entry() throws Exception {
     db.prepareDbUnit(getClass(), "update_user.xml");
     createDefaultGroup();
+    userIndexer.index();
```

Comment:
All these "userIndexer.index();" (see following ones in next tests) are useless just after db insertion, please remove them.